src/parser/parsers/heroes.py

```diff
@@ -47,6 +47,15 @@ def run(self):
 
                 all_hero_stats[hero_key] = json_utils.sort_dict(hero_stats)
 
+                # Analysis - if more analyses of these similar nature are made, they can be moved to another file; for now it runs when parsed
+                # Confirm the following stats are all 1
+                # If any are not 1, the wiki should likely get this information added either to the hero infobox template or elsewhere, and the stat removed from the list below
+                multipliers = ['TechRange', 'ReloadSpeed', 'TechDuration', 'ProcBuildUpRateScale']
```

Comment:
This comment maybe needs to be clearer. "analyses of these similar nature are made", similar to what?
And what is the significance of the current multipliers? Are they the only ones that have no level scaling?

Reply:
These stats arent on the infobox because fhey’re always 1 at base. No benefit to saying a hero has 100% base range for ex.